mcu_base_class.py

```diff
@@ -16,20 +16,22 @@ class McuBaseClass():
 
     def __init__(self, device: mcu_device.McuDevice):
         self.MsgT = ["", ""]
-        self.Tracks = [mcu_track.McuTrack() for i in range(0)] # empty array, since "import typing" is not supported
+        # empty array, since "import typing" is not supported
+        self.Tracks = [mcu_track.McuTrack() for i in range(0)]
 
-        self.Shift = False #indicates that the shift button is pressed
+        self.Shift = False  # indicates that the shift button is pressed
         self.MsgDirty = False
 
         self.FirstTrack = 0
         self.FirstTrackT = [0, 0]
 
-        self.FreeCtrlT = [0 for x in range(mcu_constants.FreeTrackCount + 1)]  # 64+1 sliders
+        self.FreeCtrlT = [0 for x in range(
```

Comment:
It does feel like the Auto PIP-8 formatting is very quick to split lines while it does not always immediately feel like they are really too long. Would it be possible to increase the line length a little bit?